tests/bluetooth/bsim_bt/edtt_ble_test_app/README.txt

```diff
@@ -0,0 +1,64 @@
+Intro
+#####
+
+This is the embedded side of the BLE conformance tests which are part of the
+EDTT (Embedded Device Test Tool).
+
+Much more info about the tool can be found in https://github.com/EDTTool/EDTT
+and in its `doc/` folder.
+
+In very short, there is 2 applications in this folder:
+1.) A controller only build of the BLE stack, where the HCI, and a few extra
+    interfaces are exposed to the EDTT.
+2.) An application which implements the test GATT services specified by BT SIG
+    in GATT_Test_Databases.xlsm, with an EDTT interface which allows the EDTT
+    tests to switch between theses.
+
+The first application is used for LL and HCI conformance tests of the
+controller.
+The second application for GATT tests.
+
+These 2 particular applications are meant to be used in a simulated environment,
+for regression, either as part of a CI system, or in workstation during
+development.
+This is due to this particular application only containing an EDTT transport
+driver for simulated targets, meant to connect to the `bsim` EDTT transport
+driver thru the EDTT bridge.
+
+How to use it
+#############
+
+First of all, add to your environment the variable EDTT_PATH poitining to the
+EDTT folder. You can do this by adding it to your `~/.bashrc`, `~/.zephyrrc`,
+or similar something like:
+```
+export EDTT_PATH=${ZEPHYR_BASE}/../tools/EDTT/
+```
+(if you add it to your .bashrc you probably won't be able to refer to
+ZEPHYR_BASE)
+
+To run these set of tests you need to compile both of these applications as any
+other Zephyr app, targeting the nrf52_bsim.
```

Comment:
Don't we need to mention BabbleSim as a prerequisite here? Otherwise users who don't have it installed won't be able to run EDTT with `nrf52_bsim`?

Reply:
mentioned as the first paragraph